queries.sql

```diff
@@ -6,4 +6,15 @@ SELECT date_of_birth FROM animals WHERE name = 'Pikachu' OR name = 'Agumon';
 SELECT name, escape_attempts  FROM animals WHERE weight_kg > 10.5;
```

Comment:
- [ ] In the queries file you need to have the following transactions, kindly do so

![image](https://user-images.githubusercontent.com/81932478/208258099-2ade60ed-3505-4223-8d65-736a12ac4e38.png)
2
![image](https://user-images.githubusercontent.com/81932478/208258186-0b8ac254-fd4b-431c-8492-689e91be651d.png)

3
![image](https://user-images.githubusercontent.com/81932478/208258272-6361bede-c14a-48a9-bcd4-45c7abd7f7c1.png)

4
![image](https://user-images.githubusercontent.com/81932478/208258382-2a149ca2-346f-4622-a317-759631e8c02b.png)